src/SFA.DAS.AdminService.Web/Views/Application/Sequence.cshtml

```diff
@@ -36,27 +36,51 @@
                             <dd class="govuk-summary-list__value">@Model.ApplicationReference</dd>
                         </div>
                     }
+                    @if (!string.IsNullOrWhiteSpace(Model.LegalName) && Model.IsWithdrawal)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">Organisation name</dt>
+                            <dd class="govuk-summary-list__value">@Model.LegalName</dd>
+                        </div>
+                    }
                     @if (!string.IsNullOrWhiteSpace(Model.Standard) && Model.SequenceNo == ApplyConst.STANDARD_SEQUENCE_NO)
                     {
                         <div class="govuk-summary-list__row">
                             <dt class="govuk-summary-list__key">Standard</dt>
                             <dd class="govuk-summary-list__value">@Model.Standard</dd>
                         </div>
                     }
-                    @if (!string.IsNullOrWhiteSpace(Model.LegalName))
+                    @if (!string.IsNullOrWhiteSpace(Model.Standard) && Model.SequenceNo == ApplyConst.STANDARD_WITHDRAWAL_SEQUENCE_NO)
                     {
                         <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">Registered name</dt>
-                            <dd class="govuk-summary-list__value">@Model.LegalName</dd>
+                            <dt class="govuk-summary-list__key">Details</dt>
+                            <dd class="govuk-summary-list__value">@Model.Standard</dd>
```

Comment:
It doesn't look like this includes the STRef - I think it should from the AC

Reply:
Changed to ST Ref